eth_test_parser/src/deserialize.rs

```diff
@@ -34,71 +34,71 @@ where
 }
 
 #[derive(Deserialize, Debug)]
-struct ByteString(#[serde(with = "serde_bytes")] pub Vec<u8>);
+pub(crate) struct ByteString(#[serde(with = "serde_bytes")] pub Vec<u8>);
 
 #[derive(Deserialize, Debug)]
 #[serde(rename_all = "camelCase")]
-struct Env {
-    current_base_fee: U256,
-    current_coinbase: H160,
-    current_difficulty: U256,
-    current_gas_limit: U256,
-    current_number: U256,
-    current_random: U256,
-    current_timestamp: U256,
-    previous_hash: H256,
+pub(crate) struct Env {
+    pub current_base_fee: U256,
+    pub current_coinbase: H160,
+    pub current_difficulty: U256,
+    pub current_gas_limit: U256,
+    pub current_number: U256,
+    pub current_random: U256,
+    pub current_timestamp: U256,
+    pub previous_hash: H256,
```

Comment:
Nit: Might be better to set these to `pub(crate)`. I'm not 100% sure what happens when you have a `pub(crate)` struct with a `pub` field, but I know `pub(crate)` items don't get added to the public generated docs.

Reply:
Yeah. Very much a nit, but it looks like it makes a bit of a difference: https://www.reddit.com/r/learnrust/comments/mezhj8/pub_versus_pubcrate_in_a_private_module/